lessons/pydata/classification_metrics/index.ipynb

```diff
@@ -0,0 +1,527 @@
+{
```

Comment:
<p>Klasifikátor vykazuje / dosahuje na této množině... (toho "mít" už je tam nějak moc)</p><p> </p><p>Tykat?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/85/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>